widgets/events/src/EventsTable.tsx

```diff
@@ -131,37 +136,49 @@ export default class EventsTable extends React.Component {
                         label="Blueprint"
                         name="blueprint_id"
                         show={
-                            _.size(data.blueprintId) !== 1 &&
-                            _.size(data.deploymentId) !== 1 &&
-                            _.size(data.nodeInstanceId) !== 1 &&
-                            _.size(data.executionId) !== 1 &&
+                            !this.sOneElementLike(data.blueprintId) &&
+                            !this.sOneElementLike(data.deploymentId) &&
+                            !this.sOneElementLike(data.nodeInstanceId) &&
+                            !this.sOneElementLike(data.executionId) &&
                             fieldsToShow.indexOf('Blueprint') >= 0
                         }
                     />
                     <DataTable.Column
                         label="Deployment"
+                        name="deployment_display_name"
+                        show={
+                            !this.sOneElementLike(data.deploymentId) &&
+                            !this.sOneElementLike(data.nodeInstanceId) &&
+                            !this.sOneElementLike(data.executionId) &&
+                            fieldsToShow.indexOf('Deployment Id') >= 0
```

Comment:
Shouldn't this be

```suggestion
                            fieldsToShow.indexOf('Deployment') >= 0
```

?

Reply:
Minor:

Btw. we could also use `includes`, which IMO has a nicer API than `indexOf`

```suggestion
                            fieldsToShow.includes('Deployment Id')
```

This applies to all uses of `fieldsToShow.indexOf` in this file :slightly_smiling_face:

Reply:
Will change, but i don't want to add more unnecessary changes in this PR.